org.kde.kst.yml

```diff
@@ -0,0 +1,99 @@
+app-id: org.kde.kst
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: kst2
+finish-args:
+  - '--share=network'
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--socket=wayland'
+  - '--filesystem=host'
```

Comment:
that is a very broad filesystem permission

Reply:
Fixed, most user will want to plot stuff exclusively from their home anyway